src/api/Main/BuisnessLayer/Shrooms.DataTransferObjects/Models/Kudos/KudosWelcomeDto.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Shrooms.DataTransferObjects.Models.Kudos
+{
+    public class KudosWelcomeDTO
+    {
+        public decimal KudosWelcomeAmount { get; set; }
+
+        public bool KudosWelcomeEnabled { get; set; }
```

Comment:
I recommend to keep consistency and name it "IsActive" instead of "Enabled" like it's already named in KudosTypes table. (It's relevant to all places where this DTO or corresponding ViewModel is used).